ENet-SAD-Simple/dataloader.lua

```diff
@@ -0,0 +1,168 @@
+--
```

Comment:
Please try not to add files from the torch repo. They should only be referenced within your own code